leaflet/templates/leaflet/admin/change_list_geo.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'admin/change_list.html' %}
+{% load leaflet_tags %}
+{% load admin_list_leaflet %}
```

Comment:
Hi,
I'm not sure anymore (didn't look at it for some time), maybe it was a mistake ? did you try to remove that line ?